chatkit-core/src/test/kotlin/com/pusher/chatkit/rooms/state/JoinedRoomTest.kt

```diff
@@ -0,0 +1,32 @@
+package com.pusher.chatkit.rooms.state
+
+import assertk.assertThat
+import assertk.assertions.isNotNull
+import com.pusher.chatkit.state.ChatkitState
+import com.pusher.chatkit.state.JoinedRoom
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.describe
+
+class JoinedRoomTest : Spek({
+
+    describe("given no rooms") {
```

Comment:
it would be also good to have tests for one given room

Reply:
👍 have added this test.